cmd/git/main.go

```diff
@@ -156,6 +158,17 @@ func runGitClone(ctx context.Context) error {
 		}
 	}
 
+	if flagValues.revision == "" && flagValues.resultFileBranchName != "" {
```

Comment:
use of `strings.TrimSpace(flagValues.revision)` and `strings.TrimSpace(flagValues.resultFileBranchName)` is a nit but possibly worth pursuing

Reply:
updated !